### ISSUE TITLE
About Rocket Mod

### ISSUE THREAD

**Issue by Maximjetfs22** [SmartlyDressedGames/Unturned-3.x-Community#1377]
There are rumors here that Rocekt Mod will no longer be supported, is that true?

**Comment by JustOneMoreBlock** [comment]
It's not a rumor.

https://github.com/RocketMod/Rocket/blob/master/Farewell.md

However, the license allows you to fork it and someone can take over.

If so, I would hope someone has some strict guidelines when providing plugins so they "don't die" and remove the need for different versions of many dlls. Basically everyone needs to conform to policy.

Alternatively, There's always uMod which also supports Unturned, however, not as prominent as RocketMod. https://umod.org/

**Comment by joeymisfit** [comment]
It's not that RocketMod isn't supported, it's that the developers decided to stop working on it, and it is open-source on GitHub so that developers can continue on. Soon enough ImperialPlugins will more than likely have a RM plugin section that will be updated as needed, just like they already offer a hyperlink at the top to the latest RM release.

**Comment by JustOneMoreBlock** [comment]
- RocketMod works.
- RocketMod isn't supported. (Discord, Forums, and plugins have been shut down.) This is a clear indication.

Hopefully, it does continue as I'm at a point of stopping Unturned altogether because Unturned breaks, RocketMod needs to be updated and at least one or twice a year everyone goes through pulling out their hair to find fixes or workarounds.

**Comment by SDGNelson** [comment]
Despite not receiving updates going forward, it will continue to be compatible with Unturned. I've written up some documentation here: [RocketMod](https://github.com/SmartlyDressedGames/U3-Docs/blob/master/RocketMod.md)

The only components that may break are individual plugins occasionally because there are always unexpected things being done.

If anyone picks up RocketMod community elements I'm open to working with them and linking from the docs. 

The biggest complication right now is probably that the individual plugins aren't compiled by Sven anymore.

**Comment by JustOneMoreBlock** [comment]
> Despite not receiving updates going forward, it will continue to be compatible with Unturned. I've written up some documentation here: [RocketMod](https://github.com/SmartlyDressedGames/U3-Docs/blob/master/RocketMod.md)
> 
> The only components that may break are individual plugins occasionally because there are always unexpected things being done.
> 
> If anyone picks up RocketMod community elements I'm open to working with them and linking from the docs.
> 
> The biggest complication right now is probably that the individual plugins aren't compiled by Sven anymore.

I am familiar with Jenkins and probably wouldn't be hard to get all the plugins compiled, however, I would want to make changes:

Example:

- I18N.West.dll (DEPRECATED)
- I18N.dll (DEPRECATED)
- I18N.v2.West.dll (DEPRECATED)
- I18N.v2.dll (DEPRECATED)
- I18N.v4.West.dll (DEPRECATED)
- I18N.v4.dll
- MySql.Data.v6.8.8.0.dll (DEPRECATED)
- MySql.Data.v6.9.5.0.dll (DEPRECATED)
- MySql.Data.v6.9.8.0.dll (DEPRECATED)
- MySql.Data.v8.0.16.dll
- System.Data.dll (DEPRECATED)
- System.Data.v2.dll
- System.Management.dll (DEPRECATED)
- System.Management.v2.dll
- System.Transactions.dll (DEPRECATED)
- System.Transactions.v2.dll

Basically keep one version of the latest version. So, we don't have different versions. MySQL should be just built-in with a config with easy variables that can easily be extended to the plugin.

If we update a specific module and requires a dll and yours would be outdated. The plugin will be forced to add the new library to compile with the new build. It won't likely break everything if it' 's something like MySQL.

**Comment by Trojaner** [comment]
We at ImperialPlugins are planning to keep maintaining free plugins and are also planning to compile them

**Comment by JustOneMoreBlock** [comment]
@Trojaner That's good to hear. Do you have an ETA?

**Comment by Trojaner** [comment]
No ETA yet, but we are trying to add them as soon as possible

**Comment by HanaForward** [comment]
I am trying to reset the plugin.
Reset plugins remain free and open source.

**Comment by HanaForward** [comment]
use mono .net 4.5 dll
and MySql.Data 6.9.12.0 with orm SqlSugar 5.0.0.10
